src/app/common/services/users/users.service.ts

```diff
@@ -12,12 +12,17 @@ const defLimit: number = 20;
 @Injectable()
 export class UsersService {
   picture = 'assets/images/mainflux-logo.png';
+  loginUrl = '/auth/login';
 
   constructor(
     private http: HttpClient,
     private router: Router,
     private notificationsService: NotificationsService,
-  ) { }
+  ) {
+    if (environment.appPrefix !== '') {
+      this.loginUrl = '/' + environment.appPrefix + '/auth/login';
```

Comment:
Probably better to do 
```
  loginUrl = (environment.appPrefix !== '') ? /auth/login : '/' + environment.appPrefix + '/auth/login';
```

BTW, maybe it would be better to have `/app` as default prefix and to remove the `/` that you have to add in the URLs